### PR TITLE
fix bug https://github.com/alibaba/Sentinel/issues/1906

### PR DESCRIPTION
**Pull request by howiekang**
### Describe what this PR does / why we need it
修复了 #1906

在Cluster客户端 进行参数校验，不是基本类型的参数 会抛出异常，不再发送到Cluster Server

### Does this pull request fix one issue?
 #1906

### CODE REVIEW

**Review by brotherlu-xcq — CHANGES_REQUESTED** [review]

File: sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java [hunk 1]
```diff
@@ -196,7 +202,7 @@ public void stop() throws Exception {
     }
 
     private boolean validRequest(Request request) {
-        return request != null && request.getType() >= 0;
+        return request != null && request.getType() >= 0 ;
```
[the remark this answers is not in the document]
Comment (howiekang) [reply sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java]: I changed it.

**Review by brotherlu-xcq — DISMISSED** [review]
LGTM

File: sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java [hunk 2]
```diff
@@ -212,6 +218,10 @@ public ClusterResponse sendRequest(ClusterRequest request) throws Exception {
         if (!validRequest(request)) {
             throw new SentinelClusterException(ClusterErrorMessages.BAD_REQUEST);
         }
+        // fix bug https://github.com/alibaba/Sentinel/issues/1906
+        if (!checkParamFlowData(request)){
```
Comment (sczyh30) [inline sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java:222]: `NettyTransportClient` is designed as the transport-layer implementation, which is not associated to the concrete request type (and content). Maybe adding the logic in `DefaultClusterTokenClient` seems better?

**Review by sczyh30 — CHANGES_REQUESTED** [review]

File: sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java (hunk 2 above)
Comment (howiekang) [reply sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java]: Moved `checkParamFlowData` to `DefaultClusterTokenClient`. The logic of `checkParamFlowData` has not changed. However, if there are other `ClusterTokenClient` implementations in the future, the `checkParamFlowData` logic cannot be reused.

**Pull request closed without merging**